src/aurora.ts

```diff
@@ -319,6 +319,14 @@ export class Aurora extends Construct {
     };
     const parameters = props.parameterGroup || props.parameters ? props.parameters : defaultParameters;
 
+    const instanceProps = {
+      performanceInsightEncryptionKey: encryptionKey,
+      performanceInsightRetention: props.performanceInsightRetention,
+      instanceType,
+      isFromLegacyInstanceProps: true,
+    };
+    const nReaders = props.instances == null ? 1 : props.instances - 1;
```

Comment:
This logic needs to be covered by UT, if it isn't already.

Also, I'm not sure that `null` and `undefined` are the same thing. Better to go with `falsey`, I think? 

```suggestion
    // default to 1 reader
    const nReaders = !props.instances  ? 1 : props.instances - 1;
```

Reply:
Looks like https://github.com/time-loop/cdk-aurora/blob/main/test/aurora.test.ts#L203 gives sufficient coverage.

Reply:
Also, must update docstring at https://github.com/time-loop/cdk-aurora/pull/85/files#diff-452005033ae4f920e11e18304773a4bf27bf15f87ec2e60e807468ed53869a4eL72